sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveCatalogs.scala

```diff
@@ -106,6 +106,31 @@ class ResolveCatalogs(val catalogManager: CatalogManager)
       val columns = u.columns.map(UnresolvedAttribute(_))
       UpdateTable(aliased, columns, u.values, u.condition)
 
+    case m @ MergeIntoTable(target, source, _, _, _) =>
```

Comment:
Why do we need to handle `MergeIntoTable` here? `target`, `source` are children of `MergeIntoTable` and will be resolved by the analyzer automatically.

Reply:
Yea, I revisited the code and remove this unnecessary logic. 

BTW, I kept some of checking v1 table logic for `MergeIntoTable` in `ResolveSessionCatalog`, but it seems not a good choice to place the v1 table check there for just checking. What do you think? Should we move the check to `CheckAnalysis` or somewhere else?

Reply:
I think we don't need the check right now as there is no implementation. We can add the check later.